go/libkb/log_send.go

```diff
@@ -529,20 +538,10 @@ func (l *LogSendContext) LogSend(statusJSON, feedback string, sendLogs bool, num
 		if mergeExtendedStatus {
 			statusJSON = l.mergeExtendedStatus(statusJSON)
 		}
-	} else {
-		kbfsLog = ""
```

Comment:
I also got rid of this else branch, as far as I can tell it doesn't actually do anything.